spec/function.dd

```diff
@@ -2054,14 +2054,12 @@ void func(T)(T x)
 {
     writeln(x);
 }
-
```

Comment:
Links:

http://dtest.dlang.io/artifact/website-5f8248914455d3d83945a3b59d88073e5ee6fc7c-82f02509ce078ef3f9b9e8813db20d59/web/spec/function.html

http://dtest.dlang.io/artifact/website-5f8248914455d3d83945a3b59d88073e5ee6fc7c-82f02509ce078ef3f9b9e8813db20d59/web/spec/attribute.html#disable

Kinda ugly, but if you say it's not worth bothering then so be it.